model-server/src/test/kotlin/org/modelix/model/server/ModelServerTestUtil.kt

```diff
@@ -37,3 +41,23 @@ fun Application.installDefaultServerPlugins() {
     install(Resources)
     install(IgnoreTrailingSlash)
 }
+
+fun runWithNettyServer(
```

Comment:
Maybe some documentation on when to use this over the default test server of ktor would be good.

Reply:
Done